projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -242,10 +246,15 @@ export class IgxInputGroupComponent extends DisplayDensityBase implements IgxInp
         private _inputGroupType: IgxInputGroupType,
         @Inject(DOCUMENT)
         private document: any,
-        private renderer: Renderer2,
-        private platform: PlatformUtil
+        private platform: PlatformUtil,
+        private cdr: ChangeDetectorRef
     ) {
         super(_displayDensityOptions);
+
+        this._subscription = this._theme$.asObservable().subscribe(value => {
+            this._theme = value as IgxInputGroupTheme;
+            this.cdr.detectChanges();
```

Comment:
this should work and with `this.cdr.markForCheck()`

Reply:
ok, does it have any performance implications?

Reply:
@wnvko 
Technically it should, however if we don't want to adjust all editing tests for the grid that use the input group, I suggest we leave it as it is. This will be called usually once only at initialization and the performance hit should be negligible.